app/models/custom_button_event.rb

```diff
@@ -2,6 +2,14 @@ class CustomButtonEvent < EventStream
   virtual_column :button_name, :type => :string
   virtual_column :automate_entry_point, :type => :string
 
+  def self.scope_for_model(user)
```

Comment:
Do we need this scope in `CustomButtonEvent? Could we just put the `super_admin_user?` in `Rbac#fulterer`?

Reply:
@kbrock `CustomButtonEvent` records should be available for super admin users only, we're requesting them through `report_data` so AFAIK we don't have any other way to enforce RBAC on them. cc @h-kataria